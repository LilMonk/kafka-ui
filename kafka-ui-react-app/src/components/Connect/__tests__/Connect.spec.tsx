import React from 'react';
import { render } from 'lib/testHelpers';
import Connect from 'components/Connect/Connect';

describe('Connect', () => {
  it('matches snapshot', () => {
    const wrapper = render(<Connect />);
    expect(wrapper.baseElement).toMatchSnapshot();
  });
});
