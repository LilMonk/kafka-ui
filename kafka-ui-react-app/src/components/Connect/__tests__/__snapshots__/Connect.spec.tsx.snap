// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect matches snapshot 1`] = `
<body>
  <div>
    <div />
  </div>
</body>
`;
